### PR TITLE
[HOTFIX] add build options for cdh 5.x

### PR DESCRIPTION
**Pull request by jongyoul**
minor fix for supporting cdh 5.x

### CODE REVIEW

**Comment by jongyoul** [comment]
@swkimme Please review and merge this PR.

File: README.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ Spark 1.2.x
 ```
 mvn clean package -Pspark-1.2 -Dhadoop.version=2.2.0 -Phadoop-2.2 -DskipTests 
 ```
+CDH 5.X
+```
+mvn clean package mvn clean package -Pspark-1.2 -Dhadoop.version=2.5.0-cdh5.3.0 -Phadoop-2.4 -DskipTests
```
Comment (swkimme) [inline README.md]: double 'mvn clean package'

**Comment by jongyoul** [comment]
@swkimme fixed it. Sorry for confusing you. It was very basic mistake.

**Comment by swkimme** [comment]
LGTM!

**Pull request merged**